sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/complexTypeCreator.scala

```diff
@@ -60,18 +60,24 @@ case class CreateArray(children: Seq[Expression]) extends Expression {
         ctx.INPUT_ROW,
         children.zipWithIndex.map { case (e, i) =>
           val eval = e.genCode(ctx)
-          eval.code + s"""
-            if (${eval.isNull}) {
```

Comment:
Just a memo.
I found that ```<exprcode>.map(_.isNull).mkString(" || ")``` is used in some places. This generates code sequence ```isNULL = false || false;```